config/routes.rb

```diff
@@ -1,10 +1,11 @@
 Rails.application.routes.draw do
+  devise_for :users
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  root 'static_pages#landing'
```

Comment:
It is not good idea to remove other people's code.
Please, add it back as root route. We will have access to users page through navigation bar.